demosplan/DemosPlanCoreBundle/Logic/Statement/StatementService.php

```diff
@@ -427,14 +427,16 @@ public function createOriginalStatement($data): Statement
         $statementCreatedEvent = $this->eventDispatcher->dispatch(new ManualOriginalStatementCreatedEvent($statement));
 
         if (null !== $statementCreatedEvent) {
```

Comment:
I think you can replace the `null !== $statementCreatedEvent` check with `$this->permissions->hasPermission('area_admin_statement_list')`. $statementCreatedEvent can't be null after dispatch. The sonarcloud issue should also be fixed with that.

Reply:
done in https://github.com/demos-europe/demosplan-core/pull/3557/commits/222c2aba0586cc287f589692e209a5c2548d6160